vars/packageBuildingPipelineDAOSTest.groovy

```diff
@@ -795,7 +795,11 @@ void call(Map pipeline_args) {
                                                    string(name: 'CI_PR_REPOS',
                                                           value: env.JOB_NAME.split('/')[1] + '@' +
                                                                  "${env.BRANCH_NAME}:${env.BUILD_ID}" +
-                                                                 ' ' + cachedCommitPragma('PR-repos'))
+                                                                 ' ' + cachedCommitPragma('PR-repos')),
+                                                    booleanParam(name: 'CI_RPM_Ubuntu20_NOBUILD',
+                                                           value: pipeline_args.get('skip-build-ubuntu-rpm', true) ||
+                                                                  (pipeline_args.get('skip-build', true) ||
+                                                                  !'ubuntu20.04' in distros))
```

Comment:
```suggestion
                                                                 ' ' + cachedCommitPragma('PR-repos')),
                                                    booleanParam(name: 'CI_RPM_el8_NOBUILD',
                                                           value: pipeline_args.get('skip-build', true) ||
                                                                  ! 'el8' in distros)),
                                                    booleanParam(name: 'CI_RPM_el9_NOBUILD',
                                                           value: pipeline_args.get('skip-build', true) ||
                                                                  ! 'el9' in distros)),
                                                    booleanParam(name: 'CI_RPM_leap15_NOBUILD',
                                                           value: pipeline_args.get('skip-build', true) ||
                                                                  ! 'leap15' in distros)),
                                                    booleanParam(name: 'CI_RPM_Ubuntu20_NOBUILD',
                                                           value: pipeline_args.get('skip-build', true) ||
                                                                  ! 'ubuntu20.04' in distros))
```

Reply:
It seems to work as expected:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/ci-daos-stack-go-build-toolchain-PR-25-master/61/parameters/
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/ci-daos-stack-go-build-toolchain-PR-25-master/61/pipeline-console/?selected-node=186